### ISSUE TITLE
Filter MF unknowns with non-ND evidence out of annotation preview/GPAD

### ISSUE THREAD

**Issue by krchristie** [geneontology/noctua#463]
Today at MGI's GO group meeting, we were discussing the presence of the unknown, i.e. root level, MF annotations in the Annotation Preview.

We were wondering if it would be possible to filter out the root level MF annotations with non-ND evidence codes, i.e. the ones made from annotons where the function involved in the process or localized to a component is not known. 
- Making these root MF annotations with non-ND evidence codes is against the GO guidelines for using root level annotations specifically indicate when a curator has looked and has been unable to find information for a specific GO aspect: http://www.geneontology.org/page/nd-no-biological-data-available
- Having all these extra MF annotations to the root makes it harder to check that the model is producing the annotations you do want, for example see the model indicated below where I want to make 11 CC annotations and 0 MF annotations:

**ID:** gomodel:595a9ee700000731 
**Name:** PMID:27914912-KRC Foxj1 targets 

Note that this would only apply to root level MF annotations with non-ND codes. We would still want to be able to make root level annotations WITH the ND code as discussed in this ticket: https://github.com/geneontology/noctua/issues/320

**Comment by edwong57** [comment]
SGD curators would love this too. In our training exercises, we have been hacking this by removing the  'molecular_function' from the annoton so that it doesn't show up in the GPAD/annotation preview, but it isn't ideal.

**Comment by balhoff** [comment]
This should be live after the next Minerva restart.

**Comment by balhoff** [comment]
@krchristie can you check that this works as you expect now? Thanks.

**Comment by kltm** [comment]
Looking for feedback from @krchristie (or @edwong57 ) before closing out.

**Comment by krchristie** [comment]
Apologies so slow to get to this, a combination of several different trips with an email rule that isn't specific enough to flag my attention for Noctua tickets I'm mentioned in.

For the model I listed above when I opened the ticket, the filtering out of MF unknowns is great, leaving only the component annotations I want.

I also tested that root MF annotations do not get filtered out using this model: 

**ID: gomodel:59a105b300000000 
Name: TEST: unknown with ND** 

Both look good to me.

thanks!
